src/goto-symex/symex_main.cpp

```diff
@@ -361,28 +360,27 @@ void goto_symext::symex_with_state(
 
   symex_threaded_step(state, get_goto_function);
   if(should_pause_symex)
-    return;
+    return state.symbol_table;
+
   while(!state.call_stack().empty())
   {
     state.has_saved_jump_target = false;
     state.has_saved_next_instruction = false;
     symex_threaded_step(state, get_goto_function);
     if(should_pause_symex)
-      return;
+      return state.symbol_table;
```

Comment:
I am not convinced this commit is actually using any copy-elision. Are the performance characteristics still the same?

Reply:
Actually I should not have included this in here, it was just drive-by cleanup. Commit now removed and instead PR-ed as #7275. Perhaps you can elaborate on that new PR?